src/components/OfflineWithFeedback.tsx

```diff
@@ -96,8 +96,8 @@ function OfflineWithFeedback({
      * This method applies the strikethrough to all the children passed recursively
      */
     const applyStrikeThrough = useCallback(
```

Comment:
Would you walk me through this?

How does `applyStrikeThrough` relate to `Tooltip`s children?

Reply:
Actually, would you point me to any `Tooltip` usage that _doesn't_ pass a single JSX child tag to it? I can't find such usage, but I'm searching naively for the `<Tooltip ` string.

Reply:
> Actually, would you point me to any Tooltip usage that doesn't pass a single JSX child tag to it? I can't find such usage, but I'm searching naively for the <Tooltip  string.

I did not find multiple children being passed explicitly to the `Tooltip` - the `applyStrikeThrough` seems to be the only case of an array being passed as it uses `Children.map`. What we've done here is, the array will be omitted if only single child exist (which is a case for all of the `Tooltip` usages).

> How does applyStrikeThrough relate to Tooltip's children?

They are used in a parent-child relation in `MenuItem` in `WorkspaceInitialPage` and `WorkspaceListPage`. It is where the the strikethrough kicks in while the app is offline.

Reply:
Would you take a look [at this helper branch](https://github.com/cubuspl42/App/commits/investigate-Tooltip-children-1/) I based on top of your PR?

I understand that you say that this code (including my top commits) is now incorrect; breaks the app in some way. Please correct me if I misunderstood you.

If I got you right, would you provide me the repro steps to observe this breakage? I guess this would show up as an exception and a console error, but any observable breakage would do. A video would also be helpful.

I thought this might be the fastest way to ensure we're on the same page.

Reply:
Sure thing! 

In order to break the app in this scenario, please follow steps in [this issue](https://github.com/Expensify/App/issues/32379). This is exactly the regression we have discussed before 👍